src/components/modules/header/HeaderWrapper.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect } from 'react';
+import { HeaderProps, Cta } from '../../../types/HeaderTypes';
+import { addRegionIdAndTrackingParamsToSupportUrl } from '../../../lib/tracking';
+import { OphanAction } from '../../../types/OphanTypes';
+import { HasBeenSeen, useHasBeenSeen } from '../../../hooks/useHasBeenSeen';
+
+export interface HeaderEnrichedCta {
+    ctaUrl: string;
+    ctaText: string;
+}
+
+export interface HeaderRenderedContent {
+    heading: string;
+    subheading: string;
+    primaryCta: HeaderEnrichedCta | null;
+    secondaryCta: HeaderEnrichedCta | null;
+}
+
+export interface HeaderRenderProps {
+    content: HeaderRenderedContent;
+}
```

Comment:
We weren't 100% sure about types here. Technically we could just use:

```ts
export interface HeaderRenderProps {
  content: HeaderContent;
}
```

But I think maybe it's nice to explicitly separate them out. Perhaps we could share `HeaderEnrichedCta` with `BannerEnrichedCta` though? At that point we could almost share `buildEnrichedCta` with the `bannerWrapper`. Unfortunately atm the `header` uses it's own `Cta` type, and not the shared one though

Reply:
We could refactor to share which wouldn't be much work, but it is technically a breaking change could lead to some temporary issues when rolling it out

Reply:
what do you think @tomrf1 ?

Reply:
So `HeaderRenderedContent` and `HeaderContent` are effectively the same? I'm fine with keeping them separate though.
I guess for the banner the 'rendered' types are more different because they can contain `JSX.Element`s.

Regarding ctas, I'm not sure how much we gain from sharing with the banner, I wouldn't let it slow you down here anyway

Reply:
Yeah makes sense - we'll keep it how it is!